pkg/kubernetes/storageclass/v1alpha1/storageclass.go

```diff
@@ -0,0 +1,110 @@
+package v1alpha1
+
+import (
+	storagev1 "k8s.io/api/storage/v1"
+)
+
+// StorageClass is a wrapper over API based
+// storage class instance
+type StorageClass struct {
+	object *storagev1.StorageClass
+}
+
+// StorageClassList holds the list of StorageClass instances
+type StorageClassList struct {
+	items []*StorageClass
```

Comment:
You should export `items` as this will be used outside of this package.

Reply:
Will push new commit